charmhelpers/core/hookenv.py

```diff
@@ -226,6 +226,13 @@ def relation_id(relation_name=None, service_or_unit=None):
         raise ValueError('Must specify neither or both of relation_name and service_or_unit')
 
 
+def departing_unit():
+    """The departing unit for the current relation hook.
+
+    Available since juju 2.8"""
```

Comment:
Please could we start to add type information for the params and return values for new functions (or if we touch old ones).  It really helps with debugging to see the intended types that a function takes and returns.

e.g. in this case:

```
:returns: the departing unit, or None if the information isn't available.
:rtype: Optional[str]
```

Reply:
done.